test/CMakeLists.txt

```diff
@@ -167,6 +167,7 @@ set(TILEDB_UNIT_TEST_SOURCES
   src/unit-lru_cache.cc
   src/unit-tile-metadata.cc
   src/unit-tile-metadata-generator.cc
+  src/unit-bytevecvalue.cc
```

Comment:
Hello! Unfortunately serialization for OR clauses isn't supported right now, this will be in the next PR.

Reply:
The smoke test doesn't run serialization, so we can add coverage on this PR.

Reply:
Resolved, by adding coverage in this PR

Reply:
I am unresolving this because since the clauses() method is used in this PR and I have support to check that we are only checking previously supported trees, I cannot ensure that the smoke test works with OR clauses in this PR. I will ensure it works in the serialization support PR.

Reply:
Can we file a shortcut item that is blocking for the release?